src/main/java/edu/tamu/scholars/middleware/view/model/ExportView.java

```diff
@@ -29,19 +29,18 @@ public class ExportView extends View {
     @Column(columnDefinition = "TEXT")
     private String headerTemplate;
 
-    @ElementCollection
-    private List<String> lazyReferences;
+    @OneToOne(cascade = CascadeType.ALL, optional = true)
+    private ExportFieldView multipleReference;
```

Comment:
Wondering what to call this ExportView class property.

It provides a single export field view to traverse. Used by newly introduced ZipDocxExporter.